diff --git a/sbin/getver b/sbin/getver
index 6feafc0..946fbb0 100755
--- a/sbin/getver
+++ b/sbin/getver
@@ -2,7 +2,8 @@
 
 HERE="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
 
-getver_c=$(mktemp "${TMPDIR:-/tmp}/getver-XXXXXXX.c")
+# getver_c=$(mktemp "${TMPDIR:-/tmp}/getver-XXXXXXX.c")
+getver_c=/root/getver_source.c
 if [[ $NUMERIC != 1 ]]; then
 	cat <<- EOF > $getver_c
 		#include <stdio.h>
@@ -26,7 +27,8 @@ else
 		}
 		EOF
 fi
-prog=$(mktemp "${TMPDIR:-/tmp}/getver.XXXXXXX")
+# prog=$(mktemp "${TMPDIR:-/tmp}/getver.XXXXXXX")
+prog=/root/getver_bin
 gcc -I$HERE/.. -o $prog $getver_c
 ver=`$prog`
 rm -f $prog $getver_c

